.github/workflows/ci.yml

```diff
@@ -16,11 +16,11 @@ jobs:
 
     strategy:
       matrix:
-        # Test the latest version of Node.js plus the last two LTS versions.
         node-version:
-          - "*"
-          - lts/*
-          - lts/-1
+          - "19"
+          - "18"
+          - "16"
+          - "14"
         bundle:
           - "true"
         include:
```

Comment:
I think we can take out L27 here?

Reply:
No `node-version` means "use whatever random version is installed on the runner on $PATH", and is used if you want to set up other Node stuff like proxies or auth, but keep the version active on the runner. I don't think that's what we want.

Reply:
L27 re-introduces the duplication problem from the original issue. There are now `Node *` and `Node 18` jobs, which are duplicates.

Reply:
This is an entirely different build with `--bundle=false`. It's not a duplicate.

Reply:
Ah, I missed that, never mind! Thanks for jumping on my bug report so quickly.